docs/faq.asciidoc

```diff
@@ -143,22 +168,59 @@ INFO      "delete_indices" action completed.
 
 You can read more on Curator here: https://www.elastic.co/guide/en/elasticsearch/client/curator/current/index.html
 
-[float]
-[[reduce-payload-size]]
-==== Reduce the payload size
-
-Large payloads coming from agents may result in "503: Request timed out waiting to be processed" messages.
-You can reduce the payload size by decreasing the `max_queue_size` in the agents.
-This will result in agents sending smaller payloads to the APM Server,
-but the requests will be more frequent.
-See the documentation for the {apm-py-ref}/configuration.html#config-max-queue-size[Python] and {apm-node-ref}/agent-api.html#max-queue-size[Node.js] agents for more information.
+If you want to delete documents that match a specific query, e.g. all documents with a given `context.service.name`,
+you can do this by sending following request against your Elasticsearch API:
+["source","sh",subs="attributes"]
+------------------------------------------------------------
+POST /apm-*/_delete_by_query
+{
+  "query": {
+    "bool": {
+      "must": [
+        {
+          "term": {
+            "context.service.name": {
+              "value": "old-service-name"
+            }
+          }
+        }
+      ]
+    }
+  }
+}
+------------------------------------------------------------
```

Comment:
You should add //CONSOLE to the next line after the dashes so that the published topic has Copy as Curl and View In Console links:

```
}
---------------------------------------------------------------
// CONSOLE
```

The doc build will add the links to the bottom of the example. Here's an example from the Beats docs. The links make it really easy for users to run the examples.

![image](https://user-images.githubusercontent.com/14206422/41991067-1a2cf7c8-79f9-11e8-8fd3-69c537e064f4.png)

Reply:
I didn't check for other examples in the doc, but we should do this whenever possible.